package.json

```diff
@@ -57,11 +58,11 @@
     "react-transition-group-plus": "^0.5.3",
     "slugify": "^1.4.6",
     "stackdriver-errors-js": "^0.8.0",
-    "web3modal": "^1.9.1",
-    "web3": "^1.3.0"
+    "web3": "^1.3.0",
+    "web3modal": "^1.9.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "HTTPS=true react-app-rewired start",
```

Comment:
I would remove this before making the PR final, as this is only required for local testing of the Safe apps

Reply:
Hi, so this PR is not ready to merge?